<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PingerWindow</class>
 <widget class="QMainWindow" name="PingerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PingerWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="pingerOut"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Хост:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hostname"/>
       </item>
       <item>
        <widget class="QPushButton" name="pingButton">
         <property name="text">
          <string>Пинг</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear">
         <property name="text">
          <string>Очистить окно</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>Остановить процесс</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>pingerOut</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
